src/promptflow/tests/executor/e2etests/test_executor_happypath.py

```diff
@@ -225,3 +239,20 @@ def test_executor_for_script_tool_with_init(self, dev_connections):
         flow_result = executor.exec_line({"input": "World"})
         assert flow_result.run_info.status == Status.Completed
         assert flow_result.output["output"] == "Hello World"
+
+
+def target_func(queue, flow_file, node_name, flow_inputs, dependency_nodes_outputs, connections, raise_ex):
```

Comment:
It would be better to use a more clear name.

Reply:
Updated.